scripts/valhalla_build_config

```diff
@@ -212,6 +213,7 @@ help_text = {
     'admin': 'Location of sqlite file holding admin polygons created with valhalla_build_admins',
     'timezone': 'Location of sqlite file holding timezone information created with valhalla_build_timezones',
     'transit_dir': 'Location of intermediate transit tiles created with valhalla_build_transit',
+    'bounding_box': 'Add comma separated bounding box values to only download transit data inside the given bounding box',
```

Comment:
Can you call this transit_bounding_box just to distinguish that it only impacts transit?

Reply:
Done!

Reply:
i would actually argue that its better off just being the command line argument and left completely out of the config. i actually hope that we keep moving more and more things out of the config and turning them into command line arguments. would you guys be ok with just not having this in the config at all?

Reply:
@kevinkreiser I would agree with you if you didn't have so many optional arguments, which have an order to be passed on the command. And honestly, your config file help_text is the best documentation for what those arguments are, as the other documentation is lacking a little bit on details.

Reply:
yeah this program really needs to get proper `--help` and other `--` arguments. the only reason it doesnt have them is because we've been too lazy to do it everywhere :(

Reply:
@kevinkreiser And you know what else it needs? Consistency on how the arguments are passed. I already found 3 different ways to pass the config file value. `-c config.json` on valhalla_build_tiles, `--config config_json` on valhalla_build_admins and valhalla_build_timezones` and just  as argv[1] `config.json` in valhalla_fetch_transit and valhalla_build_transit. Of course, I'm not a programmer, and there may be very good reasons to have it that way, but it's kind of confusing.

Reply:
`-c` and `--config` are the short and long versions respectively that is what we should shoot for. also having programs take positional arguments (when we are lazy) is dumb in my opinion. we should just be diligent and only do the short/long combo and be done. this also forces us to document the arguments in the correct way.